src/consensus/pbft/pbft.h

```diff
@@ -174,7 +170,7 @@ namespace pbft
         0,
         pbft_network.get(),
         &message_receiver_base);
-      LOG_INFO_FMT("PBFT setup for self with id: {}", local_id);
+      LOG_INFO_FMT("PBFT setup for local_id with id: {}", local_id);
```

Comment:
Minor: This message is weirdly worded now, perhaps just "PBFT setup for local_id: {}"?